Mathlib/Analysis/Convex/EGauge.lean

```diff
@@ -32,7 +32,7 @@ section SMul
 /-- The Minkowski functional for vector spaces over normed fields.
 Given a set `s` in a vector space over a normed field `𝕜`,
 `egauge s` is the functional which sends `x : E`
-to the infimum of `‖c‖₊` over `c` such that `x` belongs to `s` scaled by `c`.
+to the infimum of `‖c‖ₑ` over `c` such that `x` belongs to `s` scaled by `c`.
 
 The definition only requires `𝕜` to have a `NNNorm` instance
```

Comment:
This comment is outdated now; please fix :-)

Reply:
Well, it wasn't yet outdated, but now it is 😁